rules/tock.bzl

```diff
@@ -77,6 +82,7 @@ tock_elf2tab = rule(
         "src": attr.label(mandatory = True, allow_single_file = True, doc = "ELF binary to convert"),
         "disable": attr.bool(default = False, doc = "Mark the application as disabled"),
         "arch": attr.string(mandatory = True, doc = "Target architecture for the ELF binary (e.g., `rv32imc`)"),
+        "flash_start": attr.int(default = 0x20040000, doc = "Application starting address in flash"),
```

Comment:
Good point, I made it mandatory.

In the future, the link address and app's linker script should probably be tied together somehow (put into the same provider?) so that it's harder to mess up.